sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -369,6 +370,22 @@ datetimePart
     | complexDateTimePart
     ;
 
+nestedFunction
+    : NESTED LR_BRACKET
+    (nestedField | nestedField COMMA nestedPath)
```

Comment:
Maybe
```suggestion
    nestedField | (nestedField COMMA nestedPath)
```

Reply:
This breaks all nested functions in the parser.